authorization/index.md

```diff
@@ -540,8 +540,20 @@ Authorization: Bearer i8hweunweunweofiwweoijewiwe
   "birthTime": ...
 }
 ```
-
+The EHR FHIR resource server validates the access token and ensures that it has 
+not expired and that its scope covers the requested FHIR resource.  The 
+resource server also validates that the URL given for the `aud` parameter is its 
+own FHIR endpoint.  The method used by the EHR to validate the access token is 
+beyond the scope of this specification but generally involves an interaction or 
+coordination between the EHR’s resource server and the authorization server.
+
+On occasion, an app may receive a FHIR resource that contains a “reference” to 
+a resource hosted on a different resource server.  The app SHOULD NOT blindly 
+follow such references and send along its access_token, as the token may be 
+subject to potential theft.   The app SHOULD either ignore the reference, or 
+initiate a new request for access to that resource.  
```

Comment:
Does "initiate a new request" here mean that the app should initiate a new *authorization* request?